tests/driver_dsp0401/README.md

```diff
@@ -12,7 +12,7 @@ The display can be controlled from the JP1 connection jumper as follows:
 * Connect `PWM` to the board `DSP0401_PARAM_PWM_DEV` pwm device. See the board periph_conf.h
 to find a valid pin and configuration.
 * Connect `VCC` to one of the board VCC pin (3.3V or 5V depending on your board.
-  Example: use 3.3V with ST Nucleo)
+  Example: use 3.3V with STM32 Nucleo-64)
```

Comment:
this replace seems odd, text was good before?

Reply:
Are you just talking of the addition of `-64` ?

Reply:
If yes, this is because the default parameters of this drivers uses the pin mapping of nucleo-64 boards. Event if [only nucleo-f411 is mentionned](https://github.com/RIOT-OS/RIOT/blob/master/drivers/dsp0401/include/dsp0401_params.h#L32) it should work with others of the same family.

Reply:
ok!

Reply:
> ok!

Care to ACK @kaspar030 ?